CONTRIBUTING.md

```diff
@@ -78,6 +78,30 @@ $ melos bootstrap
 - [amplify_analytics_plugin_interface](https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_analytics_plugin_interface)
 - [amplify_analytics_pinpoint](https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_analytics_pinpoint)
 
+### Platform Setup
+
+Some platforms require additional setup. See below for details.
+
+#### Linux
+
+`libsecret` and `glib` are required for some plugins. They can be installed on Debian based linux distributions such as Ubuntu by running the command below.
+
+```sh
+sudo apt-get update && sudo apt-get install libsecret-1-dev libglib2.0-dev
+```
+
+**VS Code Remote Container**
+
+[VS Code Remote Containers](https://code.visualstudio.com/docs/remote/containers) can be used for linux development. `/devcontainer` contains a Dockerfile that will install the required dependencies.
+
+Note: To run tests for secure_storage, start the tests in a dbus session.
```

Comment:
I spent a bit of time trying to find a way to start a dbus daemon when the container starts up, but didn't have any luck. I might look into it again in the future, but for now this seems to work fine.

Reply:
Idk if this works in this setup but maybe worth a try: https://github.com/dnys1/amplify-dart/blob/main/packages/secure_storage/amplify_secure_storage_dart/tool/test.sh

It's been a while since I tested this too

Reply:
Thanks! I quickly tested that a similar script will work in a remote container. I need to re-enable the linux tests in CI so I will open a separate PR that enables the tests in CI and updates this.